benchmarking/prototype_parameterized_template.py

```
@@ -0,0 +1,613 @@
+    workspace = Workspace.load(workspace_path)
+    if not workspace.analyses:
+        raise RuntimeError(f"No analyses found in {workspace_path}")
+    requested = set(args.analysis or [])
+    analyses = [
+        analysis
+        for analysis in workspace.analyses
+        if (not requested and analysis.name != "L_ch0") or analysis.name in requested
+    ]
+    missing = requested - {analysis.name for analysis in analyses}
+    if missing:
+        raise KeyError(f"Unknown analyses: {sorted(missing)}")
```

Ruff (0.16.0)

[warning] Avoid specifying long messages outside the exception class

(TRY003)